ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -64,6 +72,11 @@ import           Ouroboros.Consensus.Shelley.Ledger.Ledger
 import           Ouroboros.Consensus.Shelley.Ledger.NetworkProtocolVersion
                      (ShelleyNodeToClientVersion (..))
 
+import           Control.DeepSeq (NFData)
```

Comment:
Please fit these in above.

I'm not sure where `Data.VMap` goes.... is it from ledger? If so, please put it in the ledger block (:sigh:).

Reply:
Moved.